app/controllers/settings_controller.rb

```diff
@@ -9,11 +9,11 @@ class SettingsController < ApplicationController
   end
 
   def index
-    @settings = Setting.live_descendants.search_for(params[:search])
+    @settings = Setting.live_descendants.search_for(params[:search]).where(nil)
```

Comment:
I think it would be better to use `all` here since [`all` now returns an ActiveRecord::Relation now in Rails 4](http://api.rubyonrails.org/classes/ActiveRecord/Scoping/Named/ClassMethods.html#method-i-all).